src/browser/squirrel-update.coffee

```diff
@@ -21,6 +21,9 @@ fileKeyPath = 'HKCU\\Software\\Classes\\*\\shell\\Atom'
 directoryKeyPath = 'HKCU\\Software\\Classes\\directory\\shell\\Atom'
 backgroundKeyPath = 'HKCU\\Software\\Classes\\directory\\background\\shell\\Atom'
 environmentKeyPath = 'HKCU\\Environment'
+openWithKeyPath = 'HKCU\\Software\\Classes\\Atom'
+filetypeKeyPath = (fileExtension) -> "HKCU\\Software\\Classes\\.#{fileExtension}\\OpenWithProgIds"
+openWithFileExtensions = ['scpt', 'applescript', 'as', 'asp', 'asa', 'aspx', 'ascx', 'asmx', 'ashx', 'bib', 'c', 'cc', 'cp', 'cpp', 'cxx', 'c++', 'cs', 'coffee', 'COMMIT_EDITMSG', 'cfdg', 'clj', 'cljs', 'csv', 'tsv', 'cgi', 'fcgi', 'cfg', 'conf', 'config', 'htaccess', 'css', 'diff', 'dtd', 'dylan', 'erl', 'hrl', 'fscript', 'f', 'for', 'fpp', 'f77', 'f90', 'f95', 'h', 'pch', 'hh', 'hpp', 'hxx', 'h++', 'go', 'gtd', 'gtdlog', 'hs', 'lhs', 'htm', 'html', 'phtml', 'shtml', 'inc', 'ics', 'ini', 'io', 'java', 'bsh', 'properties', 'js', 'htc', 'jsp', 'json', 'ldif', 'less', 'lisp', 'cl', 'l', 'lsp', 'mud', 'el', 'log', 'logo', 'lua', 'markdown', 'mdown', 'markdn', 'md', 'mk', 'wiki', 'wikipedia', 'mediawiki', 's', 'mips', 'spim', 'asm', 'm3', 'cm3', 'moinmoin', 'm', 'mm', 'ml', 'mli', 'mll', 'mly', 'mustache', 'hbs', 'pas', 'p', 'patch', 'pl', 'pod', 'perl', 'pm', 'php', 'php3', 'php4', 'php5', 'ps', 'eps', 'dict', 'plist', 'scriptSuite', 'scriptTerminology', 'py', 'rpy', 'cpy', 'python', 'r', 's', 'rl', 'ragel', 'rem', 'remind', 'rst', 'rest', 'rhtml', 'erb', 'erbsql', 'rb', 'rbx', 'rjs', 'rxml', 'sass', 'scss', 'scm', 'sch', 'ext', 'sh', 'ss', 'bashrc', 'bash_profile', 'bash_login', 'profile', 'bash_logout', 'slate', 'sql', 'sml', 'strings', 'svg', 'i', 'swg', 'tcl', 'tex', 'sty', 'cls', 'text', 'txt', 'utf8', 'text/plain', 'TEXT', 'sEXT', 'ttro', 'textile', 'toml', 'xhtml', 'xml', 'xsd', 'xib', 'rss', 'tld', 'pt', 'cpt', 'dtml', 'xsl', 'xslt', 'vcf', 'vcard', 'vb', 'yaml', 'yml', 'nfo', 'g', 'vss', 'd', 'e', 'gri', 'inf', 'mel', 'build', 're', 'textmate', 'fxscript', 'lgt', 'cfm', 'cfml', 'dbm', 'dbml', 'dist', 'dot', 'ics', 'ifb', 'dwt', 'g', 'in', 'l', 'm4', 'mp', 'mtml', 'orig', 'pde', 'rej', 'servlet', 's5', 'tmp', 'tpl', 'tt', 'xql', 'yy']
```

Comment:
This array seems pretty long, what do you think about storing this in a `.json` file and just doing:

```coffee
openWithFileExtensions = require './win32-open-file-extensions'
```